packages/spectral/biome.jsonc

```diff
@@ -48,5 +48,26 @@
         }
       }
     }
-  }
+  },
+  "overrides": [
+    {
+      "include": ["src/types/**/*"],
+      "linter": {
+        "rules": {
+          "nursery": {
+            "noRestrictedImports": {
+              "level": "error",
+              "options": {
+                "paths": {
```

Comment:
A bit silly but AFAIK there is no `pattern` option here, just `paths`?

Reply:
It was added in 2.2.0 of biome https://biomejs.dev/linter/rules/no-restricted-imports/#patterns